modules/rosa-sts-creating-a-cluster-with-customizations.adoc

```diff
@@ -0,0 +1,224 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-with-customizations.adoc
+
+[id="rosa-sts-creating-cluster-customizations_{context}"]
+= Creating a cluster with STS using customizations
+
+When you create an OpenShift cluster that uses the AWS Security Token Service (STS), you can customize your installation interactively. When you run `rosa create cluster --interactive` at cluster creation time, you are presented with a series of interactive prompts that enable you to customize your deployment.
+
+You can also select one of the following modes using the `--mode` option with the `rosa cluster create` subcommands:
```

Comment:
`rosa cluster create` should be `rosa create cluster`. However, this command does not (yet) have the `--mode` option.

Reply:
Maybe instead of stating the command, we can just say something like "There are two CLI modes for deploying the cluster with STS..."  or something like that.

Reply:
Thank you both for your comments. I have subsequently updated the line to state:

"There are two `rosa` CLI modes for deploying a cluster with STS:"